src/bootstrap.js

```diff
@@ -4,9 +4,79 @@
 
 /* eslint-disable no-unused-vars */
 
+const { utils: Cu } = Components;
+
+Cu.import("resource://gre/modules/Services.jsm");
+
 function startup({webExtension}, reason) {
-  webExtension.startup().then(() => {
+  Services.telemetry.registerEvents("lockbox", {
+    "startup": {
+      methods: ["startup"],
+      objects: ["addon", "webextension"],
+    },
+    "iconClick": {
+      methods: ["click"],
+      objects: ["firstrun","manage"],
+      extra_keys: ["fxauid"],
+    },
+    "signIn": {
+      methods: ["render"],
+      objects: ["signInPage"],
+    },
+    "confirmPW": {
+      methods: ["render"],
+      objects: ["confirmPWPage"],
+    },
+    "setupDone": {
+      methods: ["render"],
+      objects: ["setupDonePage"],
+      extra_keys: ["fxauid"],
+    },
+    "itemAdding": {
+      methods: ["itemAdding"],
+      objects: ["addItemForm"],
+      extra_keys: ["fxauid"],
+    },
+    "itemUpdating": {
+      methods: ["itemUpdating"],
+      objects: ["updatingItemForm"],
+      extra_keys: ["fxauid"],
+    },
+    "itemDeleting": {
+      methods: ["itemDeleting"],
+      objects: ["updatingItemForm"],
+      extra_keys: ["fxauid"],
+    },
+    "itemSelected": {
+      methods: ["itemSelected"],
+      objects: ["itemList"],
+      extra_keys: ["fxauid"],
+    },
+    "addClick": {
+      methods: ["addClick"],
+      objects: ["addButton"],
+      extra_keys: ["fxauid"],
+    },
+    "itemAdded": {
+      methods: ["itemAdded"],
+      objects: ["addItemForm"],
+      extra_keys: ["itemid","fxauid"],
+    }
```

Comment:
I think this is missing trailing comma, and some other ESLint errors (but file may need a rebase anyways):

```
/home/travis/build/mozilla-lockbox/lockbox-extension/src/bootstrap.js
  19:27  error    A space is required after ','  comma-spacing
  63:28  error    A space is required after ','  comma-spacing
  64:6   error    Missing trailing comma         comma-dangle
```